frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
```

Comment:
Name of the service should also reflect the target. In this case it could for example be `BackendApi`. Also the API as class name could be spelled with camelCase: `Api`.